bindgen/clang.rs

```diff
@@ -970,6 +970,11 @@ impl Cursor {
             })
         }
     }
+
+    /// Is this cursor's referent a namespace that is inline?
+    pub(crate) fn is_inline_namespace(&self) -> bool {
+        unsafe { clang_Cursor_isInlineNamespace(self.x) != 0 }
```

Comment:
According to https://docs.rs/clang-sys/1.8.1/clang_sys/fn.clang_Cursor_isInlineNamespace.html, this function is available in libclang 9.0 and later.

Since https://github.com/rust-lang/rust-bindgen/pull/2932, bindgen only supports libclang 9.0 or later.

Reply:
Thanks for saving me the time of checking it :)